README.md

```diff
@@ -1,2 +1,48 @@
 # grafink
-Repository hosting GSoC 2020 development on graphs
+Grafink is a spark ETL job to load data into Janusgraph.
+
+### Compiling from source
+
+```
+sbt compile
+```
+
+### Code format
+
+This project uses scalafmt to format code. For formatting code:
+
+```
+sbt scalafmt        // format sources
+sbt test:scalafmt   // format test sources
+sbt sbt:scalafmt    // format.sbt source
+```
+
+### Running unit tests
+
+```
+sbt test
+```
+
+### Creating Assembly jar
+
+```
+sbt assembly
+```
+
+### Creating Distribution
+
+```
+sbt dist
+```
+
+The above creates a deployable zip file `grafink-<version>.zip`. The contents of the zip file are:
+
+  - conf/application.conf  // Modify this config file according to the job requirements.
+  - grafink assembly jar   // The main executable jar for running spark job.
+  - bin/start.sh           // The main executable script that user can invoke to start the job.
```

Comment:
If I unzip it, I have only:

```bash
grafink julien$ ls target/universal/grafink-0.1.0-SNAPSHOT/
total 0
drwxr-xr-x    4 julien  staff   128 25 May 10:54 bin
drwxr-xr-x  172 julien  staff  5504 25 May 10:54 lib

grafink julien$ ls target/universal/grafink-0.1.0-SNAPSHOT/bin/
total 56
-rwxr-xr-x  1 julien  staff  10693 25 May 10:36 grafink
-rw-r--r--  1 julien  staff  14780 25 May 10:36 grafink.bat
```
i.e. the files above are missing

Reply:
fixed